app/middleware/getPharmacies.js

```diff
@@ -3,6 +3,14 @@ const log = require('../lib/logger');
 const dedupe = require('../lib/dedupePharmacies');
 const constants = require('../lib/constants');
 
+function logZeroResults(res) {
```

Comment:
In terms of each bit of middleware doing _one thing well_ and _should only have one reason to change_ (™) it would make sense for it to be in it's own middleware - getting and logging are not the same. Not sure what the performance overhead of adding another middleware and if you decided not to I wouldn't argue.